packages/post-rector/src/Rector/NameImportingPostRector.php

```diff
@@ -55,6 +55,10 @@ public function enterNode(Node $node): ?Node
             return null;
         }
 
+        if ($this->shouldSkip($this)) {
+            return null;
+        }
```

Comment:
What about ignoring other PostRectors? I guess there should be different place to add the skipper to.

Reply:
What would you suggest? Add this check to every PostRector?

Reply:
It should be in their parent, as the `AbstractRector` handle this in one place too

Reply:
And then call parent::enterNode on every PostRector?

Reply:
Not sure about exact implementation, I'd have to look closer when I have more time.

Thr implementation should work for any post Rector rule, and be at 1 place. That's the goal here.

Reply:
Will try to figure something out, but without the inheritance hell.

Reply:
Thank you